ietf/doc/templatetags/ietf_filters.py

```diff
@@ -389,6 +389,16 @@ def expires_soon(x,request):
 def startswith(x, y):
     return str(x).startswith(y)
 
+
+@register.filter(name='removesuffix', is_safe=True)
```

Comment:
There's a warning in the [Django documentation for filters and auto-escaping](https://docs.djangoproject.com/en/2.2/howto/custom-template-tags/#filters-and-auto-escaping) about the `is_safe` flag and removing characters:

> Be careful when deciding if your filter really does leave safe strings as safe. If you’re removing characters, you might inadvertently leave unbalanced HTML tags or entities in the result. For example, removing a > from the input might turn <a> into <a, which would need to be escaped on output to avoid causing problems. Similarly, removing a semicolon (;) can turn &amp; into &amp, which is no longer a valid entity and thus needs further escaping. Most cases won’t be nearly this tricky, but keep an eye out for any problems like that when reviewing your code.

I don't think this affects your current use, but it might be a concern if someone uses the filter differently in the future. If you're not relying on the effects of `is_safe`, might be "safer" to drop it. If you are, it'd be best to add a comment with a warning.